### ISSUE TITLE
Fatal Exception: java.lang.NullPointerException on MessageAdapter.java line 376

### ISSUE THREAD

**Issue by ckbhodge** [apptentive/apptentive-android#123]
I'm using v3.1.1 and just saw this crash from one of our users:

> 
> Date: 2016-07-07T13:51:31Z
> OS Version: 4.4.2
> Device: XL-103Q
> RAM Free: 10.2%
> Disk Free: 35.9%

```
Fatal Exception: java.lang.NullPointerException
       at com.apptentive.android.sdk.module.messagecenter.view.MessageAdapter$1.run(MessageAdapter.java:376)
       at android.os.Handler.handleCallback(Handler.java:733)
       at android.os.Handler.dispatchMessage(Handler.java:95)
       at android.os.Looper.loop(Looper.java:136)
       at android.app.ActivityThread.main(ActivityThread.java:5372)
       at java.lang.reflect.Method.invokeNative(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:515)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:970)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:786)
       at dalvik.system.NativeStart.main(NativeStart.java)
```

Is this a known issue?

**Comment by skykelsey** [comment]
Thanks @ckbhodge we'll take a look.

**Comment by skykelsey** [comment]
This will be fixed in the forthcoming v3.2.0

**Comment by skykelsey** [comment]
Hi @grndvl1, have you tried with the latest release (3.2.1)?
